packages/pubsub/src/iam.js

```diff
@@ -69,7 +69,7 @@ var util = require('util');
  */
 function IAM(pubsub, id) {
   var config = {
-    baseUrl: pubsub.defaultBaseUrl_,
+    baseUrl: 'pubsub.googleapis.com',
```

Comment:
I was looking through the code as it is today and I believe `defaultBaseUrl_` is always `pubsub.googleapis.com`. IIRC when we determine the base url, we store that as `baseUrl_`.

I just assumed we didn't support a local sandbox here, but if that's not the case I think it would be easy enough to support.